apps/files/src/components/FileUpload.vue

```diff
@@ -12,6 +12,7 @@ import Mixins from '../mixins'
 
 export default {
   props: {
+    item: { type: String, required: true },
```

Comment:
Does it need to be in file upload as well? Doesn't really seem to be used anywhere in this case.